chart/templates/server-ide-configmap.yaml

```diff
@@ -104,6 +104,34 @@ options:
 
 defaultIde: "code"
 defaultDesktopIde: "code-desktop"
+
+clients:
+  vscode:
+    defaultDesktopIDE: "code-desktop"
+    desktopIDEs: ["code-desktop"]
+    installationSteps: [
+      "Click <b>Allow</b> in the dialog by your browser.",
```

Comment:
**suggestion**: Sounds safe to use `text-sm` here if possible since we're already using this text size in the workspace-related element above.